kube-client/src/config/file_config.rs

```diff
@@ -574,14 +593,28 @@ users:
     client-certificate-data: aGVsbG8K
     client-key-data: aGVsbG8K
 "#;
-        let file = tempfile::NamedTempFile::new().expect("create config tempfile");
-        fs::write(file.path(), config_yaml).unwrap();
-        let cfg = Kubeconfig::read_from(file.path())?;
```

Comment:
I think it's ok to swap out this test for a text test. There's already a tempfile test in config/mod.rs.